src/util/clickTracking.js

```diff
@@ -0,0 +1,154 @@
+const DATA_ATTR = 'clicktrack';
+
+function cleanTrackingUrl() {
+	// TODO: investigate and refactor to support this feature
+
+	/*
+	// remove tracking information from the url to prevent bad link copying
```

Comment:
yeah, this is part of the clickTracking script in Meetup Classic, but obviously we don't have particular tracking-URL support in the new platform, and I imagine that might be a little ways off. I didn't want to delete the code but also didn't want to try to get it working without any purpose/application. I suspect it's something that we would need to make an Investigation plan for, so maybe that's the best next step.